Conformity/Cloud-Account/gcp-tf-cloud-account-connector/IAM/role.tf

```diff
@@ -0,0 +1,40 @@
+
+# Create Conformity Custom Role
+resource "google_project_iam_custom_role" "cloud_one_conformity_access" {
+  role_id     = "CloudOneConformityAccess"
+  title       = "Cloud One Conformity Access"
+  description = "Project level Custom Role for Cloud One Conformity access"
+  stage       = "ALPHA"
```

Comment:
> Do we need to set it as `ALPHA`?
> 
> 
> 
> From the documentation, it seems like it will pull the stage info from the project: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project_iam_custom_role

Good catch, its optional so we don't actually need it... I will remove

Reply:
Done!